bitcoin_transfer/spend_your_coin.md

```diff
@@ -83,60 +83,65 @@ Do you remember the main questions? **From where, to where and how much?**
 Constructing the **TxIn** and adding it to the transaction is the answer to the "from where" question.  
 Constructing the **TxOut** and adding it to the transaction is the answer to the remaining ones.
 
-The donation address of this book is: [1KF8kUVHK42XzgcmJF4Lxz4wcL5WDL97PB](https://blockchain.info/address/1KF8kUVHK42XzgcmJF4Lxz4wcL5WDL97PB)  
-This money goes into my "Coffee and Sushi Wallet" that will keep me fed and compliant while writing the rest of the book.  
-If you succeed in completing this challenge you will be able to find your contribution amongst the **Hall of the Makers** on [http://n.bitcoin.ninja/](http://n.bitcoin.ninja/) \(ordered by generosity\).
+> The donation address of this book is: [1KF8kUVHK42XzgcmJF4Lxz4wcL5WDL97PB](https://blockchain.info/address/1KF8kUVHK42XzgcmJF4Lxz4wcL5WDL97PB)  
+This money goes into Nicolas' "Coffee and Sushi Wallet" that will keep him fed and compliant while writing the rest of the book.  
+If you succeed in completing this challenge on the MainNet you will be able to find your contribution among the **Hall of the Makers** on [http://n.bitcoin.ninja/](http://n.bitcoin.ninja/) \(ordered by generosity\).
 
+To get the our MainNet address:
 ```cs
 var hallOfTheMakersAddress = new BitcoinPubKeyAddress("1KF8kUVHK42XzgcmJF4Lxz4wcL5WDL97PB");
 ```
 
-If you are working on the testnet, send the testnet coins to any testnet address.
+Or if you are working on the TestNet, send the TestNet coins to [our TestNet address](https://live.blockcypher.com/btc-testnet/address/mzp4No5cmCXjZUpf112B1XWsvWBfws5bbB/).
 
 ```cs
-var hallOfTheMakersAddress = BitcoinAddress.Create("mzp4No5cmCXjZUpf112B1XWsvWBfws5bbB");
+var hallOfTheMakersAddress = BitcoinAddress.Create("mzp4No5cmCXjZUpf112B1XWsvWBfws5bbB", Network.TestNet);
```

Comment:
NBitcoin is intelligent enough to know it is testnet, but it is better practice to specify it.